src/redux/configureStore.js

```diff
@@ -0,0 +1,4 @@
+import {createStore} from 'redux'
+import rootReducer from "./reducers"
```

Comment:
Inconsistent with the existing coding styles

Reply:
Fixed.